tests/install_testbed/openstack-config

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2012, Red Hat, Inc.
+# Alan Pevec <apevec@redhat.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+import iniparse
+import sys 
+
+parameter = value = None
+def usage():
+   sys.stderr.write(sys.argv[0] +
+                    " --set|--del config_file section [parameter] [value]\n")
+   sys.exit(1)
+
+try:
+   mode = sys.argv[1]
+   if mode not in ('--set', '--del'):
+       usage()
+   cfgfile = sys.argv[2]
+   section = sys.argv[3]
+   try:
+       parameter = sys.argv[4]
+       value = sys.argv[5]
+   except IndexError:
+       if mode == '--set':
+           usage()
+   else:
+       if mode == '--del':
+           sys.stderr.write("A value should not be specified with --del\n")
```

Comment:
This file was added to the project because we need it, but if you look at the copyright is was written by another person ;-)

Anyway, if refers that with set key, you provide a value (--set key value), but with delete key, only the key is provided.